avocado_vt/plugins/vt_init.py

```diff
@@ -270,6 +270,8 @@ def initialize(self):
                                      help_msg=help_msg,
                                      default='/tmp')
 
+            settings.merge_with_configs()
```

Comment:
Do you think we can add some sort of unit tests to catch this issue in the future? So far this behavior was broken and fixed twice, once in September 2020 and once here. Perhaps it is a rarely tested path and might benefit from a simple unit test to alarm other breaking future merges.

Reply:
Sure, I will figure it out and force-pushed here.